Automattic/wp-calypso
# Clean up the manage/all-domains feature flag

#### Changes proposed in this Pull Request

* The manage all domains screen is here to stay so no need to hide it behind feature flag. Let's clean it up

#### Testing instructions

* Open up a site specific domain management screen and then try the all domains management screen. Everything should work as it did before.
